synapse/lib/stormtypes.py

```diff
@@ -1722,6 +1751,9 @@ def __init__(self, runt, vdef, path=None):
             'merge': self._methViewMerge,
         })
 
+    def __len__(self):
```

Comment:
Having the "length" of a view implicitly be it's number of layers when in all other aspects it behaves mostly like a dict is wrong.  one could always use $view.layers.size()

Reply:
removed